README.md

```diff
@@ -40,7 +40,7 @@ Examples:
 
 Examples for common distros:
 
-- ArchLinux: `pacman -S texlive-most pandoc p7zip`
+- ArchLinux: `pacman -S ruby p7zip haskell-pandoc texlive-core texlive-fontsextra texlive-fontsrecommended texlive-latexextra`
```

Comment:
- Please remove `ruby` since those are the requirements for manual generation cf. https://github.com/noraj/OSCP-Exam-Report-Template-Markdown?tab=readme-ov-file#manual not for the `osert.rb` helper script.
- `textlive-core` is an old alias for [texlive-basic](https://archlinux.org/packages/extra/any/texlive-basic/)?
- Rather than putting a long list of texlive dependencies, why not putting the [texlive](https://archlinux.org/groups/x86_64/texlive/) group since it contaisn essentially those dependencies + a few extra (cf. https://wiki.archlinux.org/title/TeX_Live).

Reply:
I removed `ruby` but is it not needed as interpreter for running `osert.rb`?
About `texlive-core`, yes, you're right. I replaced also it by `texlive-basic` in the PR.
About the last bullet, I would avoid to put the entire texlive group just to not install elements that are not related to osert usage (so, to prevent wasted space on the final user endpoint).